src/HomePage/components/AddMovie/AddMovie.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import './AddMovie.css'
+
+export function AddMovie() {
+    const [state, setState] = React.useState({
+        title: "Add your movie",
```

Comment:
NIT: this is better to be empty string here and 'Add your movie ' is better to be a placeholder attribute for input field